src/metaopt/algo/space.py

```diff
@@ -401,10 +384,11 @@ def interval(self, alpha=1.0):
         """Return a tuple of possible values that this categorical dimension
         can take.
 
-        Awkward name.
+        .. warning:: This method makes no sense for categorical variables. Use
+           ``self.categories`` instead.
 
         """
-        return self.categories
+        raise RuntimeError("Categories are not ordered. Use ``self.categories`` instead")
```

Comment:
I find `ordered` confusing in this context. I would simply say that Categories have no interval.